poetry.lock

```diff
@@ -1,4 +1,4 @@
-# This file is automatically @generated by Poetry 1.5.1 and should not be changed by hand.
+# This file is automatically @generated by Poetry 1.7.1 and should not be changed by hand.
```

Comment:
I am not sure we should be doing this change, I don't really know what are the side effects of it

Reply:
I would say that unfortunately yes.

Reply:
CI/CD uses v1.5.1 exactly.

Reply:
Ok. I will downgrade my poetry and regenerate the file.